tensorflow/core/util/mkl_types.h

```diff
@@ -112,6 +115,7 @@ namespace tensorflow {
 #define TENSOR_FORMAT_NHWC MKL_TENSOR_FORMAT_NHWC
 #define TENSOR_MAX_DIMS MKLDNN_MAX_NDIMS
 #define GET_USR_MEM_PRIM_DESC(src) src.GetUsrMemDesc()
+#define BN_FLAGS mkldnn::normalization_flags
```

Comment:
This is redundant with line 26. Please delete one. (And please keep the one for v0 below in the same relative order.)

Reply:
Done. Pls take a look.